build/int.cloudbuild.yaml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-timeout: 3600s
+timeout: 7200s
```

Comment:
We can split into multiple build stages if tests can be run in parallel ([example](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/build/int.cloudbuild.yaml))

Reply:
I was just checking if we have a timeout issue. But I think in this case it is a problem with AloyDB APIs. They are throwing `internal error` on deleting cluster majority of the run. I will change timeout back to `3600`. Create and verify stages are always successful.